pom.xml

```diff
@@ -633,7 +633,7 @@
                 <plugin>
                     <groupId>org.codehaus.mojo</groupId>
                     <artifactId>exec-maven-plugin</artifactId>
-                    <version>1.5.0</version>
+                    <version>1.6.0</version>
```

Comment:
❔ what caused the need for the upgrade?

Reply:
I was having some issue with the Solr server not starting. I honestly don't remember now, but it works.